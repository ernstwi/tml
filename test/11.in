init foo
init bar
foo -> bar
    guard { x - y == 5 }
    sync { a }
    sync { b }
    reset { x }
    guard { x > 5 }
    reset { y }
    sync { c }
