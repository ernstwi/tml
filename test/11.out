Semantic error: 2 initial states
Semantic error: foo -> bar: 2 guards
Semantic error: foo -> bar: 3 syncs
Semantic error: foo -> bar: 2 resets
